tests/helm/__init__.py

```diff
@@ -0,0 +1,16 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
I think these files should be move to [`charts/tests/`](https://github.com/apache/airflow/tree/master/chart/tests) directory. It is very confusing that the tests are in this directory as this directory mainly contains tests for Airflow. The new contributor will have trouble finding these tests.

Reply:
@mik-laj I created a `chart_tests` directory as many parts of breeze assume we don't have the chart in the docker image.